backend/docs/api/workflows.openapi.yaml

```diff
@@ -0,0 +1,425 @@
+      requestBody:
+        description: Contains the definition of the job to be started.
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: '#/components/schemas/InputParameter'
+        required: true
+      responses:
+        "201":
+          description: Workflow created successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                  name:
+                    type: string
+        "400":
+          $ref: '#/components/responses/InvalidRequestError'
+        "404":
+          $ref: '#/components/responses/NotFoundError'
+  /api/v1/workflow/{name}/batch:
+    post:
+      summary: Start a batch of workflows
+      description: |
+        Starts a new workflow given by the name path-parameter.
+      operationId: Start Batch Workflows
+      parameters:
+      - name: name
+        in: path
+        description: Workflow identifier.
+        required: true
+        schema:
+          type: string
+      requestBody:
+        description: Contains the definition of the job to be started.
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: array
+                items:
+                  $ref: '#/components/schemas/InputParameter'
+        required: true
+      responses:
+        "201":
+          description: Workflows created successfully
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    id:
+                      type: string
+                    name:
+                      type: string
+                    error:
+                      type: string
+        "400":
+          $ref: '#/components/responses/InvalidRequestError'
+        "404":
+          $ref: '#/components/responses/NotFoundError'
+  /api/v1/workflow/{name}/{id}:
+    get:
+      summary: Gets the workflow status for the given id.
+      description: |
+        Gets the workflow status for the given workflow name and id provided
+        by the path parameters.
+      operationId: Workflow Status
+      parameters:
+      - name: name
+        in: path
+        description: Workflow identifier.
+        required: true
+        schema:
+          type: string
+      - name: id
+        in: path
+        description: Job identifier
+        required: true
+        schema:
+          type: string
+      responses:
+        "200":
+          description: Successful query
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JobStatus'
+        "404":
+          $ref: '#/components/responses/NotFoundError'
```

Comment:
Is this a duplicate endpoint of `/api/v1/jobs/{id}`?

Reply:
@merlin-northern maybe you know more about this one (I think you introduced `api/v1/jobs/{id}`)?